test/e2e/utils/api/expect.js

```diff
@@ -1,10 +1,29 @@
 import { $, text as textTaiko } from 'taiko'
 
-export const expectExists = async ({ text = null, selector = null }) => {
-  if (!text && !selector) throw new Error('One between text and selector is required')
+export const getSelection = async ({ text = null, selector = null, itemSelector = null }) => {
```

Comment:
why selector and itemSelector? 
can you just pass the button as selector?
see where it's used, and use $ to be used as selector

Reply:
removed